src/plugins/vis_type_metric/public/components/metric_vis_component.tsx

```diff
@@ -199,6 +202,6 @@ export class MetricVisComponent extends Component<MetricVisComponentProps> {
       const metrics = this.processTableGroups(this.props.visData);
       metricsHtml = metrics.map(this.renderMetric);
     }
-    return <div className="mtrVis">{metricsHtml}</div>;
+    return <div className="mtrVis visualization">{metricsHtml}</div>;
```

Comment:
1. Should `<VisualizationNoResults />` be used here?
2. Is the `visualization` class name necessary? I thought that was already included higher up the tree in `VisualizationContainer`, or do we need it twice?